commit 6c0065175f847fc517026abf1e04f0a7680c7d2c
Author: Thomas J. Moore <darktjm@gmail.com>
Date:   Fri Apr 17 17:04:32 2020 -0500

    Don't ask gog to sort games, as that modifies "preferred" sort order

diff --git a/src/website.cpp b/src/website.cpp
index a83fe44..2b6bc8f 100644
--- a/src/website.cpp
+++ b/src/website.cpp
@@ -132,7 +132,7 @@ std::vector<gameItem> Website::getGames()
 
     do
     {
-        std::string response = this->getResponse("https://www.gog.com/account/getFilteredProducts?hiddenFlag=" + std::to_string(iHidden) + "&isUpdated=" + std::to_string(iUpdated) + "&mediaType=1&sortBy=title&system=&page=" + std::to_string(i));
+        std::string response = this->getResponse("https://www.gog.com/account/getFilteredProducts?hiddenFlag=" + std::to_string(iHidden) + "&isUpdated=" + std::to_string(iUpdated) + "&mediaType=1&system=&page=" + std::to_string(i));
         std::istringstream json_stream(response);
 
         try {
@@ -279,10 +279,7 @@ std::vector<gameItem> Website::getGames()
     } while (!bAllPagesParsed);
     std::cerr << std::endl;
 
-    if (Globals::globalConfig.bIncludeHiddenProducts)
-    {
-        std::sort(games.begin(), games.end(), [](const gameItem& i, const gameItem& j) -> bool { return i.name < j.name; });
-    }
+    std::sort(games.begin(), games.end(), [](const gameItem& i, const gameItem& j) -> bool { return i.name < j.name; });
 
     return games;
 }
@@ -621,7 +618,7 @@ std::vector<wishlistItem> Website::getWishlistItems()
 
     do
     {
-        std::string response(this->getResponse("https://www.gog.com/account/wishlist/search?hasHiddenProducts=false&hiddenFlag=0&isUpdated=0&mediaType=0&sortBy=title&system=&page=" + std::to_string(i)));
+        std::string response(this->getResponse("https://www.gog.com/account/wishlist/search?hasHiddenProducts=false&hiddenFlag=0&isUpdated=0&mediaType=0&system=&page=" + std::to_string(i)));
         std::istringstream response_stream(response);
 
         try {

